src/components/Theme/src/Theme.vue

```diff
@@ -56,27 +56,28 @@ export default {
 	},
 	computed: {
 		styles() {
-			const { colors, fonts } = this;
+			const { colors, fonts, fontWeights } = this;
 
 			return {
+				'--color-background': colors.background,
+				'--color-text': colors.text,
+				'--color-heading': colors.heading,
+				'--color-primary': colors.primary,
+				'--color-secondary': colors.secondary,
```

Comment:
Since we're erring on the side of not adding necessary variables, I would not add a secondary color. There's no consistent use case for it throughout Website, in fact it doesn't define one. More often than not, outline button color and text or title color are the same. It's also a very specific definition - the color for secondary/outline buttons, not a bigger pattern.

Reply:
We could immediately use it for things like buttons and links. While we label the colors in website currently as solid and outline buttons -- there's a benefit to abstract those colors to be useable for more than buttons. There's also precedent as it's a pretty common convention to have a secondary color in the core set of colors in many systems that allow color customizations:

* https://getbootstrap.com/docs/5.1/customize/color/
* https://m3.material.io/styles/color/the-color-system/key-colors-tones

We currently aren't able to leverage the secondary button color we allow people to customize in the profile without a lookup and passing it in. I don't think at this point we should be exposing explicit and inconsistent methods for styling the components through a combination of generic tokens, props, variants, and now patterns as it creates additional confusion about how the theming system works while it's still in a state flux. 

I believe the safest way for us to expand our theming options is through the global / core tokens for colors, fonts, and shape. From there we can internally change and refine how they are used and applied to the components until we're at a point of exposing maximum flexibility for styling the components in a consistent way.

Reply:
How does the abstraction it line up? Is `colors.primary` always the solid button color?

If so, having a `primary` and `secondary` Maker color could cause some confusion on Website, where `--primary-color` always means brand color and `--secondary-color` is deprecated on older sites. Whereas `--maker-primary-color` means the color of the primary CTA which could be different than the brand color.